### ISSUE TITLE
Executable Size and Distribution

### ISSUE THREAD

**Issue by devloop0** [poi-the-machine/the-machine#7]
As the release of the first build approaches, there are some kinks to be worked out. Of course the source code will lie here on Github, but there is the issue of executable size. Currently, the application relies on Emgu CV and might rely on NAudio too (as of now, it is bundled, but is not used, but its contribution to the executable size is minor). Currently, the executable size is around 800 MBs with all of the dll dependencies. Compressed with 7z, that goes down to 120 MB.
There are a few options on how to go about this:
        1) We can bundle the executable in the source repository compressed with 7z. However, the downside would be that it would significantly increase the "git clone" download size and bundles the source and the executable together.
        2) We can create a separate repository for all of the executables, but this raises the other problem that executables are generally not on Git.
        3) A hosting site (Mega)? This would probably be the best would require more work. Maybe a Github.io site could be used as well.

Of course, these are my opinions on how we should proceed. If there are any other options or opinions, I would love to hear them. Any help is greatly appreciated. The good news is that this should be the extent of the executable size until speech processing is added.

TL;DR Help with executable size.

**Comment by Xalaxis** [comment]
I would say keep the builds in the main branch.  If you are going to work
on the project's source you are likely going to either want a full 'clone',
or just download the files you need manually (or even work using gits
built in editor)

**Comment by devloop0** [comment]
Okay, thanks for the suggestion. For the source and the executable, I'll just create a separate folder called windows and keep everything in there. 
I'll mark this as closed.

**Comment by Xalaxis** [comment]
I also suggest a MEGA mirror, just in case somebody doesn't want to download it from Github for whatever reason (but definitely keep the Github copy up because it *is* part of our code, and you can download individual files).

**Comment by devloop0** [comment]
I've already created an account. Once I have a stable WiFi connection, I'll upload a testing copy to make sure everything is okay.
